docs/book/src/testing/unit-testing.md

```diff
@@ -104,3 +104,13 @@ Example `Forc.toml` for contract above:
 ```toml
 {{#include ../../../../examples/multi_contract_calls/caller/Forc.toml:multi_contract_call_toml}}
 ```
+
+## Running Tests in Parallel or Serially
+
+By default, tests are run in parallel using threads. Note that each test is assigned their own storage so different tests does not invalidate a shared state by running in parallel.
```

Comment:
```suggestion
By default, all unit tests in your project are run in parallel. Note that this does not lead to any data races in storage because each unit test has its own storage space that is not shared by any other unit test.
```